### PR TITLE
ARROW-11406: [CI][C++] Fix ccache caching on Travis-CI

### PR DESCRIPTION
**Pull request by pitrou**

### CODE REVIEW

**Comment by pitrou** [comment]
Builds on Travis-CI:
* first build (cold cache): https://travis-ci.com/github/pitrou/arrow/builds/214951977
* second build (hot cache): https://travis-ci.com/github/pitrou/arrow/builds/214956140

File: .travis.yml [hunk 1]
```diff
@@ -143,11 +148,13 @@ script:
   #   /home/travis/.travis/functions: line 109: ulimit: core file size: cannot modify limit: Operation not permitted
   - |
     ulimit -c unlimited || :
+  - du -d2 -h $DOCKER_VOLUME_PREFIX
```
Comment (kou) [inline .travis.yml:151]: Do we need this?
We can see similar information by ccache statistics report: https://travis-ci.com/github/pitrou/arrow/jobs/477453957#L1533

**Review by kou — APPROVED** [review]
+1

File: .travis.yml (hunk 1 above)
Comment (pitrou) [reply .travis.yml:151]: No, that was to debug until I got it to work.

**Comment by pitrou** [comment]
Updated build: https://travis-ci.com/github/pitrou/arrow/builds/215044886

**Pull request closed without merging**